conference/fares.py

```diff
@@ -191,6 +194,19 @@ def set_early_bird_fare_dates(conference, start_date, end_date):
     early_birds.update(start_validity=start_date, end_validity=end_date)
 
 
+def disable_early_bird_fares(conference):
+    early_birds = Fare.objects.filter(
+        conference=conference,
+        code__regex=FARE_CODE_REGEXES["types"][FARE_CODE_TYPES.EARLY_BIRD],
+    )
+    assert (
+        early_birds.count()
+        == len(FARE_CODE_VARIANTS) * len(FARE_CODE_GROUPS)
+        == 3 * 5
```

Comment:
the 3 * 5 part of the assertion is, for various reasons, likely to fail on production.
yes, it's perfect for dev, and dev is modeled after production, but it's more likely going to be 3*3 :)